<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepairDataMainWindow</class>
 <widget class="QMainWindow" name="RepairDataMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(51, 51, 51);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="Menu" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(150, 150, 150);
background-color: rgb(51, 51, 51);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="ButtonRepairDepartment">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Rockwell</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.372, stop:0.17017 rgba(0, 27, 72, 255), stop:0.371371 rgba(0, 0, 0, 255), stop:0.668669 rgba(4, 0, 5, 255), stop:1 rgba(105, 12, 137, 255));
font: 10pt &quot;Rockwell&quot;;</string>
         </property>
         <property name="text">
          <string>Редагувати відділення</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRepairSpeciality">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.372, stop:0.171853 rgba(8, 72, 0, 255), stop:0.371371 rgba(0, 0, 0, 255), stop:0.668669 rgba(4, 0, 5, 255), stop:1 rgba(0, 27, 72, 255));
font: 10pt &quot;Rockwell&quot;;
</string>
         </property>
         <property name="text">
          <string>Редагувати спеціальність</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRepairGroup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Rockwell&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.372, stop:0.17282 rgba(72, 71, 0, 255), stop:0.371371 rgba(0, 0, 0, 255), stop:0.668669 rgba(4, 0, 5, 255), stop:1 rgba(0, 72, 15, 255));</string>
         </property>
         <property name="text">
          <string>Редагувати групу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRepairStudent">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Rockwell&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.372, stop:0.17282 rgba(72, 43, 0, 255), stop:0.371371 rgba(0, 0, 0, 255), stop:0.668669 rgba(4, 0, 5, 255), stop:1 rgba(72, 71, 0, 255));</string>
         </property>
         <property name="text">
          <string>Редагувати студента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRepairLecturer">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Rockwell&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0.865, x2:0.702, y2:1, stop:0 rgba(0, 72, 65, 255), stop:1 rgba(0, 0, 0, 255));</string>
         </property>
         <property name="text">
          <string>Редагувати викладача</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonRepairSubject">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Rockwell&quot;;
background-color: qlineargradient(spread:pad, x1:1, y1:0.865, x2:0.702, y2:1, stop:0 rgba(0, 72, 65, 255), stop:1 rgba(0, 0, 0, 255));</string>
         </property>
         <property name="text">
          <string>Редагувати предмет</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
